docs/preview/connected-registry/connected-registry-error-codes.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Connected registry error code reference
+description: Details about error codes shown in the statusDetails property of a connected registry resource. For each error, possible solutions are listed.
+ms.topic: article
+ms.date: 04/13/2021
+ms.author: jeburke
+author: jaysterp
+---
+
+# Connected registry error code reference
+
+This article helps you troubleshoot error codes you might encounter in the `StatusDetails` property of a connected registry.
+
+## Status Details Format
+
+When a connected registry has a connection state of `Unhealthy`, this indicates there is a critical error on the instance running on-premises. You may reference the `StatusDetails` property of the connected registry resource to view the corresponding error.
```

Comment:
To provide related context, can you also briefly describe what other connection state is?

Reply:
I see you mentioned the transition from `unhealthy` to `online`. Is there any other state?

Reply:
Sure let me add more context here.

Reply:
Currently, our only states are Online, Offline, and Unhealthy.